Makefile

```diff
@@ -1,16 +1,18 @@
+export PATH := $(PWD)/bin:$(PWD)/venv/bin:$(PATH)
 
 REBUILD_FLAG =
 
 .PHONY: all
 all: venv test
 
-.PHONY: venv
 venv: .venv.touch
-	tox -e venv $(REBUILD_FLAG)
+	rm -rf venv
+	virtualenv venv --python python2.7
+	pip install -r requirements_dev.txt
```

Comment:
You either want a virtualenv or you want tox, not both.

What error are you running into that requires this change?  (I'm guessing it's our internal system packaging of `tox` and `tox2`).  Ping me in irc and I'll show you the precedent that's been established for other packages that want to do this.